backend/config/keys_dev.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  mongoURI:
+    // 'mongodb+srv://godstime:eWC2hZqacjFP8KZH@cluster0-7achi.mongodb.net/test?retryWrites=true&w=majority',
+    "mongodb+srv://godstime:9OTpPdtiwoL7uEMQ@cluster0-rorws.mongodb.net/test?retryWrites=true&w=majority",
```

Comment:
this should be in `.gitignore` ? or is this for everyone to use mongo atlas in the cloud